test/factories/locations.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :location do
+    va_accepted Faker::Boolean.boolean
```

Comment:
In the other factories, the call to `Faker` is wrapped in `{ }`:  https://github.com/OperationCode/operationcode_backend/blob/master/test/factories/users.rb#L4

Can you confirm if these are needed or not?

Reply:
@hpjaj I'm reasonably confident we don't need the `{}`'s, all the tests pass when using the `create(:code_school)` method, where we see the Factory is defined using the same syntax.